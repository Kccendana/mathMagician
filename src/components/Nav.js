import { Link } from 'react-router-dom';
import logo from '../assets/calc.svg';
import './nav.css';

const Nav = () => (
  <nav className="navbar">
    <img src={logo} alt="logo" className="logo" />
    <ul>
      <li><Link to="/">Home</Link></li>
      <li><Link to="/calculator">Calculator</Link></li>
      <li><Link to="/quote">Quote</Link></li>
    </ul>
  </nav>
);

export default Nav;
