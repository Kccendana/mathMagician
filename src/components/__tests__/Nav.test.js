import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import Nav from "../Nav"; // Adjust the path

describe('Nav component', () => {
    it('should match the snapshot', () => {
        const { container } = render(
          <BrowserRouter>
            <Nav />
          </BrowserRouter>);
        expect(container).toMatchSnapshot();
    });
});