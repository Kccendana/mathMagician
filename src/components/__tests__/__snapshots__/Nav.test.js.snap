// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component should match the snapshot 1`] = `
<div>
  <nav
    class="navbar"
  >
    <img
      alt="logo"
      class="logo"
      src="calc.svg"
    />
    <ul>
      <li>
        <a
          href="/"
        >
          Home
        </a>
      </li>
      <li>
        <a
          href="/calculator"
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          href="/quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
